packages/vm/Cargo.toml

```diff
@@ -45,10 +44,12 @@ sha2 = "0.9.1"
 thiserror = "1.0"
 wasmer = { git = "https://github.com/wasmerio/wasmer", rev = "59431f", default-features = false, features = ["jit", "singlepass"] }
 wasmer-engine = { git = "https://github.com/wasmerio/wasmer", rev = "59431f" }
+wasmer-middlewares = { git = "https://github.com/wasmerio/wasmer", rev = "59431f" }
```

Comment:
Bleeding edge, following master.

We need to wait for them to tag before we can cut/publish a release based on this.